storage/reads/datatypes/storage_common.proto

```diff
@@ -165,6 +167,36 @@ message ReadResponse {
     DataType data_type = 2;
   }
 
+  message MultiFloat {
+    repeated double values = 1;
+  }
+  message MultiInteger {
+    repeated int64 values = 1;
+  }
+  message MultiUnsigned {
+    repeated uint64 values = 1;
+  }
+  message MultiBoolean {
+    repeated bool values = 1;
+  }
+  message MultiString {
+    repeated string values = 1;
+  }
+  message AnyPoints {
+    oneof data {
+      MultiFloat float_v = 1;
+      MultiInteger integer_v = 2;
+      MultiUnsigned unsigned_v = 3;
+      MultiBoolean boolean_v = 4;
+      MultiString string_v = 5;
+    }
+  }
+
+  message MultiPointsFrame {
+    repeated sfixed64 timestamps = 1;
+    repeated AnyPoints value_arrays = 2 [(gogoproto.nullable) = false];
+  }
```

Comment:
New MultiPointsFrom cc @jsternberg

Reply:
Naming feedback but I like this:

```
message FloatValues {
     repeated double values = 1;
}
message IntegerValues {
     repeated int64 values = 1;
}
message UnsignedValues {
     repeated uint64 values = 1;
}
message BooleanValues {
     repeated bool values = 1;
}
message StringValues {
     repeated string values = 1;
}
message AnyPoints {
     oneof data {
       FloatValues floats = 1;
       IntegerValues integers = 2;
       UnsignedValues unsigned = 3;
       BooleanValues booleans = 4;
       StringValues strings = 5;
     }
}
```

Reply:
Updated to this. Feels a bit stutter-y but full words are better than the `_v` stuff.